certified-connectors/EXPOCAD/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4316 @@
+            "description": "The rate plan name",
+            "x-ms-summary": "The rate plan name",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "description": "A status for the rented booth",
+            "x-ms-summary": "A status for the rented booth",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "comment",
+            "in": "query",
+            "description": "A comment for the rented booth",
+            "x-ms-summary": "A comment for the rented booth",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "description": "Rents the specified booth in the supplied event."
+      }
+    },
+    "/{ClientName}/booths/unrent": {
+      "put": {
+        "tags": [
+          "Booths"
+        ],
+        "summary": "Unrent a Booth",
+        "operationId": "Booths_UnRentBooth",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "description": "Organization Account use for EXPOCAD",
+            "in": "path",
+            "name": "ClientName",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "EXPOCAD Account"
+          },
+          {
+            "name": "boothNumber",
+            "in": "query",
+            "description": "The booth number of the booth",
+            "x-ms-summary": "The booth number of the booth",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "databaseName",
+            "in": "query",
+            "description": "The database name of the EXPOCAD event",
+            "x-ms-summary": "The database name of the EXPOCAD event",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "description": "Unrents the specified booth in the supplied event."
+      }
+    },
+    "/{ClientName}/booths/hold": {
+      "put": {
+        "tags": [
+          "Booths"
+        ],
+        "summary": "Holds the specified booth",
+        "description": "EXPOCAD allows for a booth to be placed on hold to either an existing Exhibitor (by Id) or to \r\n            a temporary Exhibitor (by Name).<br /> Choose one parameter or the other -  'ExhibitorId' or 'ExhibitorName'.<br /><br />\r\n            If both paramerters are supplied, the hold will default to the Exhibitor ID.",
```

Comment:
@mikefactorial Please remove the extra spacing in the value

Reply:
This has been resolved in the latest commit